src/OhTenPHP/Website/SiteBundle/Services/TwitterService.php

```diff
@@ -47,9 +42,11 @@ public function __construct(Twitter $client, Pool $cache)
     /**
      * Returns the latest tweets from the defined user timeline.
      *
+     * @param string $screenName The twitter account for which to get the timeline.
+     *
      * @return mixed
      */
-    public function getLatestTweets()
+    public function getLatestTweets($screenName)
```

Comment:
If we configure the screen name, I would inject it via the constructor instead, we don't have use cases for different screenNames